packages/core/src/SlotMixin.js

```diff
@@ -23,7 +26,9 @@ const SlotMixinImplementation = superclass =>
     }
 
     connectedCallback() {
+      // @ts-ignore checking this in case we pass LitElement, found no good way to type this...
```

Comment:
Is there a reason this mixin is not dependent on LitEl?

Reply:
Because it could be that you pass HTMLElement instead. So it's either HTMLElement or LitElement, but you can't do something like `@type {LitElement | HTMLElement}` properly :( but feel free to experiment and see if you can find a better way